batch/batch/front_end/front_end.py

```diff
@@ -2231,7 +2231,7 @@ async def delete_batch(request: web.Request, _, batch_id: int) -> web.Response:
     return web.Response()
 
 
-@routes.get('/batches/{batch_id}')
+@routes.get('/batches/{batch_id}', name='batch')
```

Comment:
Small request - can we name this something like 'batch_list_page' or 'batches_page'?

Reply:
Oops, I missed the `{batch_id}`. This already looks more correct than my suggestions, but maybe something like `batch_page` or `batch_details` might be nicer?